WebGPU API docs (#24202)
mdn/content: files/en-us/web/api/htmlcanvaselement/getcontext/index.md

      object representing a three-dimensional rendering context. This context is only
      available on browsers that implement [WebGL](/en-US/docs/Web/API/WebGL_API)
      version 2 (OpenGL ES 3.0). {{experimental_inline}}
    - `"webgpu"`, which will create a
      {{domxref("GPUCanvasContext")}} object representing a three-dimensional

>       rendering context for GPU render pipelines. This context is only available on browsers that implement [WebGPU](/en-US/docs/Web/API/WebGPU_API).

Reviewer: "for GPU render pipelines" could technically apply to WebGL as well. Not sure what the best way to differentiate it here is. Maybe something like this?

```suggestion
      rendering context with a more modern API. This context is only available on browsers that implement [WebGPU](/en-US/docs/Web/API/WebGPU_API).
```

That should be true for at least a few years, right? 😅

Author: I've just put "WebGPU" instead of "GPU" — does that work for you? On MDN, we are really quite hardcore about not using text that could become dated.

Reviewer: Sounds fine to me!
